content/writing/moving-to-a-four-day-week.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Moving to a four-day week"
+date: 2021-10-02
+intro: It's the future!
+---
+
+About three months ago we decided to trial a four-day week, at no loss of pay, for all members. We reviewed it yesterday and it easily passed as a permanent change to our employment statuses. In this post, we'll outline how we did it, what we discovered in our trial and why the four-day week is an important step for any organisation.
```

Comment:
1. I'd make the when we reviewed it different as we didn't do it yesterday now :)
2. I'd paragraph at "In this post"